granulate_utils/metadata/databricks_client.py

```diff
@@ -56,21 +59,18 @@ def get_webui_address() -> Optional[str]:
             raise DatabricksJobNameDiscoverException(f"Failed to get Databricks webui address {properties=}") from e
         return f"{host}:{DEFAULT_WEBUI_PORT}"
 
-    def get_job_name(self) -> Optional[str]:
+    def extract_relevant_metadata(self) -> Optional[Dict[str, str]]:
         # Retry in case of a connection error, as the metrics server might not be up yet.
         start_time = time.monotonic()
         while time.monotonic() - start_time < DATABRICKS_JOBNAME_TIMEOUT_S:
             try:
-                if cluster_metadata := self._cluster_all_tags_metadata():
-                    name = self._get_name_from_metadata(cluster_metadata)
-                    if name:
-                        self.logger.debug("Found name in metadata", job_name=name, cluster_metadata=cluster_metadata)
-                        return name
-                    else:
-                        self.logger.debug("Failed to extract name from metadata", cluster_metadata=cluster_metadata)
-                        return None
+                if cluster_all_props := self._cluster_all_tags_metadata():
+                    self.logger.info(
+                        "Successfully got relevant cluster tags metadata", cluster_all_props=cluster_all_props
+                    )
+                    return cluster_all_props
                 else:
-                    # No job name yet, retry.
+                    # No environment metadata yet, retry.
                     time.sleep(RETRY_INTERVAL_S)
             except DatabricksJobNameDiscoverException:
                 self.logger.exception("Failed to get Databricks job name")
```

Comment:
These log messages should also be changed right?

Reply:
67bb152

Reply:
There's another log line under it also :)